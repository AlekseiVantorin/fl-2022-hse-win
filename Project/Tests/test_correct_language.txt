6
Qa
Qb
Qc
Qprev
Qn
Qend
3
a
b
c
7
a
b
c
_
A
B
C
Qa
1
Qend
16
Qa a : Qb A N
Qa B : Qn B N
Qb a : Qb a N
Qb B : Qb B N
Qb b : Qc B N
Qc b : Qc b N
Qc C : Qc C N
Qc c : Qprev C P
Qprev C : Qprev C P
Qprev b : Qprev b P
Qprev B : Qprev B P
Qprev a : Qprev a P
Qprev A : Qa A N
Qn B : Qn B N
Qn C : Qn C N
Qn _ : Qend _ N
