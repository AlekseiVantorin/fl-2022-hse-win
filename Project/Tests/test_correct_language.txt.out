$ python turing.py -f Tests/test_correct_language.txt -i "aaabbbccc" -v -k 6
_____________SOLUTION______________
State      Configuration Position   
Qa         _|a|a|a|b|b|b|c|c|c|_   1
Qb         _|A|a|a|b|b|b|c|c|c|_   2
Qb         _|A|a|a|b|b|b|c|c|c|_   3
Qb         _|A|a|a|b|b|b|c|c|c|_   4
Qc         _|A|a|a|B|b|b|c|c|c|_   5
Qc         _|A|a|a|B|b|b|c|c|c|_   6
